modules/preview-react/information-highlight/lib/modelHook.ts

```diff
@@ -1,10 +1,10 @@
 import {createModelHook} from '@workday/canvas-kit-react/common';
 
-export type Variant = 'emphasis' | 'caution' | 'attention';
+export type Variant = 'informational' | 'caution' | 'attention';
```

Comment:
Just curious, was this meant to be done in updating those variants?

Reply:
Also, I know this just one hook but, we can put this hook into a folder named `hooks`

Reply:
originally i think i did it in case there was going to be high/low emphasis options for info highlight, so it wouldnt become confusing to read. I didnt change it back, but in the specs there are emphases so i guess it worked out